altdoc/about.md

```diff
@@ -0,0 +1,38 @@
+## Developers
+
+The primary developer of the upstream Polars project is Ritchie Vink
+([@ritchie46](https://github.com/ritchie46)). This R package is
+maintained by [@eitsupi](https://github.com/eitsupi) and [Etienne Bacher](https://github.com/etiennebacher). The original R package was created by Søren Welling
+([@sorhawell](https://github.com/sorhawell)).
+
+Development occurs on Github: https://github.com/pola-rs/r-polars
+
+Consider joining our [Discord](https://discord.com/invite/4UfP5cfBE7)
+(subchannel) for additional help and discussion.
+
+**'Polars'** is a query engine/syntax for multiple languages. The R specific implementation
+can be referred to as the R API and is derived from the Rust API.
+
+The **polars** package for R is published under the MIT license.
+
+## MIT License
+
+Copyright (c) 2023 polars authors (polars the R package)
```

Comment:
I suppose we can use the include short code.

Reply:
That's not a qmd file, just md. We could convert it to qmd but that seems overkill just to have the include short code.

Reply:
The file extension has nothing to do with whether the shortcode works or not.

Reply:
Like:
https://github.com/etiennebacher/altdoc/blob/2b331663e84bdebac0212d9c7beb25ae8381b09d/R/import_readme.R#L69-L72

Reply:
But isn't that only for quarto websites? I have tried `{{< include LICENSE.md >}}` but that didn't work.

Reply:
My bad, that's right. I had forgotten that this is Mkdocs.

Use this:
https://facelessuser.github.io/pymdown-extensions/extensions/snippets/

Reply:
Done